docs/getting-started-guides/ubuntu.md

```diff
@@ -35,53 +35,71 @@ Kubernetes Deployment On Bare-metal Ubuntu Nodes
 
 - [Introduction](#introduction)
 - [Prerequisites](#prerequisites)
-    - [Starting a Cluster](#starting-a-cluster)
-        - [Make *kubernetes* , *etcd* and *flanneld* binaries](#make-kubernetes--etcd-and-flanneld-binaries)
-        - [Configure and start the Kubernetes cluster](#configure-and-start-the-kubernetes-cluster)
-        - [Deploy addons](#deploy-addons)
-        - [Trouble Shooting](#trouble-shooting)
+- [Starting a Cluster](#starting-a-cluster)
+    - [Download binaries](#download-binaries)
+    - [Configure and start the kubernetes cluster](#configure-and-start-the-kubernetes-cluster)
+    - [Test it out](#test-it-out)
+    - [Deploy Addons](#deploy-addons)
+    - [Trouble Shooting](#trouble-shooting)
 
 ## Introduction
 
-This document describes how to deploy Kubernetes on ubuntu nodes, including 1 Kubernetes master and 3 Kubernetes nodes, and people uses this approach can scale to **any number of nodes** by changing some settings with ease. The original idea was heavily inspired by @jainvipin 's ubuntu single node work, which has been merge into this document.
+This document describes how to deploy kubernetes on ubuntu nodes, 1 master and 3 nodes involved
+in the given examples. You can scale to **any number of nodes** by changing some settings with ease.
+The original idea was heavily inspired by @jainvipin 's ubuntu single node
+work, which has been merge into this document.
 
 [Cloud team from Zhejiang University](https://github.com/ZJU-SEL) will maintain this work.
 
 ## Prerequisites
 
-*1 The nodes have installed docker version 1.2+ and bridge-utils to manipulate linux bridge*
+1 The nodes have installed docker version 1.2+ and bridge-utils to manipulate linux bridge.
```

Comment:
How about making it list by adding dot after numbers (see https://guides.github.com/features/mastering-markdown/#examples tab Lists)